flagsmith/provider_test.go

```diff
@@ -0,0 +1,62 @@
+package flagsmith_test
+
+import (
+	"github.com/Flagsmith/flagsmith-go-api-client"
+	"github.com/Flagsmith/terraform-provider-flagsmith/flagsmith"
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	"os"
+	"strconv"
+	"testing"
+)
+
+var testAccProtoV6ProviderFactories = map[string]func() (tfprotov6.ProviderServer, error){
+	"flagsmith": providerserver.NewProtocol6WithError(flagsmith.New("")()),
+}
+
+func mustHaveEnv(t *testing.T, name string) {
+	if os.Getenv(name) == "" {
+		t.Fatalf("%s environment variable must be set for acceptance tests", name)
+	}
+}
+func testAccPreCheck(t *testing.T) {
+	mustHaveEnv(t, "FLAGSMITH_MASTER_API_KEY")
+	mustHaveEnv(t, "FLAGSMITH_FEATURE_NAME")
+	mustHaveEnv(t, "FLAGSMITH_ENVIRONMENT_KEY")
+	mustHaveEnv(t, "FLAGSMITH_ENVIRONMENT_ID")
+	mustHaveEnv(t, "FLAGSMITH_FEATURE_ID")
+}
+
```

Comment:
Is it intentional to have blank space between certain functions and not others?

Reply:
Well, yes to sort of group them